### DISCUSSION TITLE
Failed to parse source map from '@antv/...' after migrate from v4 to v5

### DISCUSSION THREAD

**Discussion by Storkki** [ant-design/ant-design#50512] [Q&A]
Hi, today I updated antd, react-scripts and react to latest versions, but after running react-scripts i got many warning next kind:

```
WARNING in ../node_modules/@antv/util/esm/path/util/segment-quad-factory.js
Module Warning (from ../node_modules/source-map-loader/dist/cjs.js):
Failed to parse source map from '\node_modules\@antv\util\esm\path\util\src\path\util\segment-quad-factory.ts' file: Error: ENOENT: no such file or directory, open '\node_modules\@antv\util\esm\path\util\src\path\util\segment-quad-factory.ts'
```

Is there way to fix it?

**Comment by MukeshBahuguna** [comment]
Hi,
Well there are multiple ways to get rid of these warnings, I will mention a few:

1. You can remove the warning by adding GENERATE_SOURCEMAP=false to your .env file (Worked the best for me :))
(CRA with Webpack 5.x might be the cause for it)

2. You can disable it inside your local yarn or npm script for e.g :
"scripts": {
    "start": "GENERATE_SOURCEMAP=false && react-scripts start",
 ...}

Hope this helps :)
